firefox-ios/Client/Frontend/Theme/ThemedTableViewCells/ThemedLearnMoreTableViewCell.swift

```diff
@@ -87,8 +85,7 @@ class ThemedLearnMoreTableViewCell: ThemedTableViewCell {
             learnMoreButton.topAnchor.constraint(equalTo: labelsStackView.bottomAnchor, constant: UX.labelsSpacing),
             learnMoreButton.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: UX.horizontalMargin),
             bottomConstraint,
-            learnMoreButton.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -UX.horizontalMargin),
-            learnMoreButton.heightAnchor.constraint(greaterThanOrEqualToConstant: UX.minimumHeight)
+            learnMoreButton.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -UX.horizontalMargin)
```

Comment:
Lets go ahead with this change based on our discussion in thread

Cc @thatswinnie @dicarobinho 

Although I would recommend adding a comment in the ticket that we will revisit this based on discussion from UX (Gabrielle) and A11Y (Anna) that we are merging this change while we go for an island approach in the future for when settings redesign happens.